kitsune/questions/api.py

```diff
@@ -381,6 +380,12 @@ def remove_tags(self, request, pk=None):
 
         return Response(status=status.HTTP_204_NO_CONTENT)
 
+    @action(methods=['POST'], permission_classes=[permissions.IsAuthenticated])
+    def auto_tag(self, request, pk=None):
+        question = self.get_object()
+        question.auto_tag()
```

Comment:
No, except errors with `.get()` returning too many things when Taggit gets itself into a race condition (which shouldn't happen).

`autotag` is actually really simple. It mostly just moves data from metadata to tags.